File: tests/fixtures/sample-js-project-positive-old/shared/utils/src/logger.ts
export const logger = {
  info: (message: string, ...args: readonly unknown[]): void => {
    console.info(`[INFO] ${message}`, ...args);
  },
  error: (message: string, ...args: readonly unknown[]): void => {
    console.error(`[ERROR] ${message}`, ...args);
  },
  warn: (message: string, ...args: readonly unknown[]): void => {
    console.warn(`[WARN] ${message}`, ...args);
  },
};

File: tests/fixtures/sample-csharp-project-negative/shared/Utils/Logger.cs
namespace Shared.Utils;

public class Logger
{
    public void Info(string message, params object[] args)
    {
        Console.WriteLine($"[INFO] {message}", args);
    }

    public void Error(string message, params object[] args)
    {
        Console.WriteLine($"[ERROR] {message}", args);
    }

    public void Warn(string message, params object[] args)
    {
        Console.WriteLine($"[WARN] {message}", args);
    }


    public static readonly Logger Instance = new Logger();
}
